integration_testing/features/coach/coach-activate-quiz.feature

```diff
@@ -1,22 +1,36 @@
-Feature: Coach activate quizzes
+Feature: Coach activates quizzes
   Coaches need to activate quizzes in order for learners to gain access and start submitting them
 
   Background:
     Given I am signed in to kolibri as coach user
-      And I am on *Coach > Quizzes* page
+      And I am on *Coach > Plan > Quizzes* page
       And I see the quiz <quiz_title>
 
   Scenario: Change the quiz status to *Active*
     When I click the quiz <quiz_title>
+    # Is this going to be re-implemented, or this can be achived just through the *Edit details* option?
```

Comment:
@indirectlylit ^^

Reply:
this is still intended to be implemented but hasn't happened yet.

mockup:

![image](https://user-images.githubusercontent.com/2367265/52291267-d7f27d00-2926-11e9-8dd3-62002edece18.png)

Reply:
Ok, will wait to do one final check!

Reply:
If the point is to use the scenario to test the feature, the latter needs to be implemented _before_ the former, right? 😕 
Or are we slating this to post 0.12.0?